drivers/ethernet/eth_stm32_hal.c

```diff
@@ -32,18 +32,112 @@ LOG_MODULE_REGISTER(LOG_MODULE_NAME);
+                                         uint32_t u32_PHYReg,
+                                         uint32_t* pu32_RegVal);
+
+#ifndef PHY_ADDR
+#define PHY_ADDR        ((uint32_t)CONFIG_ETH_STM32_HAL_PHY_ADDRESS)
+#endif
+
+#ifndef PHY_BSR
+#define PHY_BSR     ((uint16_t)0x0001U)
+#endif
+
+#ifndef PHY_LINKED_STATUS
+#define PHY_LINKED_STATUS       ((uint16_t)0x0004U)
+#endif
+
+#if defined(CONFIG_SOC_SERIES_STM32H7X)
+/**
+ * Definitions for ethernet driver of all others STM32 except STM32H7
+ */
+
+#define GET_TX_DESC_LIST_PTR(ps_heth)     (&(((ETH_HandleTypeDef*)ps_heth)->TxDescList))
+
+#define GET_FIRST_DMA_TX_DESC(ps_heth)                ((ETH_DMADescTypeDef*)(((ETH_HandleTypeDef*)ps_heth)->Init.TxDesc))
+#define GET_FIRST_DMA_RX_DESC(ps_heth)                ((ETH_DMADescTypeDef*)(((ETH_HandleTypeDef*)ps_heth)->Init.RxDesc))
+
+#define IS_ETH_DMATXDESC_OWN(ps_dma_tx_desc)          (((ETH_DMADescTypeDef*)ps_dma_tx_desc)->DESC3 & ETH_DMATXNDESCRF_OWN)
+#define GET_DMA_TX_BUFFER1_ADDR(ps_dma_tx_desc)       (((ETH_DMADescTypeDef*)ps_dma_tx_desc)->DESC0)
+
+#define ETH_RXBUFNB                     ETH_RX_DESC_CNT
+#define ETH_TXBUFNB                     ETH_TX_DESC_CNT
+
+#define ETH_MEDIA_INTERFACE_MII         HAL_ETH_MII_MODE
+#define ETH_MEDIA_INTERFACE_RMII        HAL_ETH_RMII_MODE
+
+#define LL_AHB1_GRP1_PERIPH_ETHMAC      LL_AHB1_GRP1_PERIPH_ETH1MAC
+#define LL_AHB1_GRP1_PERIPH_ETHMACTX    LL_AHB1_GRP1_PERIPH_ETH1TX
+#define LL_AHB1_GRP1_PERIPH_ETHMACRX    LL_AHB1_GRP1_PERIPH_ETH1RX
+#define LL_AHB1_GRP1_PERIPH_ETHMACPTP   LL_AHB1_GRP1_PERIPH_ETH1MAC
+
+#define ETH_DMA_TRANSMIT_TIMEOUT        ( 20U )
+
+static HAL_StatusTypeDef ETH_RD_PHY_REG( ETH_HandleTypeDef* ps_heth,
+                                         uint32_t u32_PHYAddr,
+                                         uint32_t u32_PHYReg,
+                                         uint32_t* pu32_RegVal) {
+    return HAL_ETH_ReadPHYRegister(ps_heth, u32_PHYAddr, u32_PHYReg, pu32_RegVal);
+}
+#else  /* CONFIG_SOC_SERIES_STM32H7X */
+/**
+ * Definitions for ethernet driver of all others STM32 except STM32H7
+ */
+
+#define GET_FIRST_DMA_TX_DESC(ps_heth)             ((ETH_DMADescTypeDef*)(((ETH_HandleTypeDef*)ps_heth)->TxDesc))
+#define GET_FIRST_DMA_RX_DESC(ps_heth)             ((ETH_DMADescTypeDef*)(((ETH_HandleTypeDef*)ps_heth)->RxDesc))
+
+#define GET_CURRENT_DMA_TX_DESC(ps_heth)           ((ETH_DMADescTypeDef*)(((ETH_HandleTypeDef*)ps_heth)->TxDesc))
+
+#define IS_ETH_DMATXDESC_OWN(ps_dma_tx_desc)      (((ETH_DMADescTypeDef*)ps_dma_tx_desc)->Status & ETH_DMATXDESC_OWN)
+#define GET_DMA_TX_BUFFER1_ADDR(ps_dma_tx_desc)   (((ETH_DMADescTypeDef*)ps_dma_tx_desc)->Buffer1Addr)
+
+static HAL_StatusTypeDef ETH_RD_PHY_REG( ETH_HandleTypeDef* ps_heth,
+                                         uint32_t u32_PHYAddr,
+                                         uint32_t u32_PHYReg,
+                                         uint32_t* pu32_RegVal) {
```

Comment:
function starting { should be in separate line

Reply:
> function starting { should be in separate line

done

Reply:
done